acceptance/bundle/deployment/bind/quality-monitor/script

```diff
@@ -0,0 +1,2 @@
+trace $CLI bundle deployment bind monitor1 "catalog.schema.table" --auto-approve
```

Comment:
What happens if you don't specify `--auto-approve`?

Reply:
this flag is not needed in this test - results are identical